* fix manual.html path, otherwise install fails
author: Miguel Dias Costa (National University of Singapore)
Adapted by Kurt Lust (University of Antwerpen)
--- Makefile.orig	2018-06-18 12:12:48.081168468 +0800
+++ Makefile	2018-06-18 12:13:02.795191541 +0800
@@ -151,7 +151,7 @@
 # install cannot work on a whole directory
 	cp -rf examples  $(INSTDIR)/share/BerkeleyGW/
 	cp -rf testsuite $(INSTDIR)/share/BerkeleyGW/
-	install manual.html $(INSTDIR)/share/BerkeleyGW/
+	cp -rf documentation $(INSTDIR)/share/BerkeleyGW/
 else
 	$(error Error: Please define installation prefix INSTDIR via 'make install INSTDIR='.)
 endif
